canvas_modules/common-canvas/src/common-properties/form/EditorForm.js

```diff
@@ -371,6 +371,9 @@ function _makeControl(parameterMetadata, paramName, group, structureDefinition,
 	const additionalText = parameter.getAdditionalText(l10nProvider);
 	const orientation = parameter.orientation;
 	const required = parameter.required;
+	if (parameter.numberGenerator) {
```

Comment:
I don't think this is the correct place to set this.  Doing it this way will end up keeping numberGenerator.label.default and/or label.resource_id in the generated form.  I think you'll want to update ParameterInfo instead and set `numberGenerator` in there possibly using a new private class.

Reply:
`l10nProvider` and `resources` information is not available in `ParameterInfo.js`.  
Added a private class in `EditorForm.js`